README.md

```diff
@@ -24,12 +22,37 @@ If you want to try out the package you will need to have a CUDA enabled GPU, `Qu
 We recommend using a conda environment `Python >= 3.7`.
 To install `CuPy` we recommend the following steps:
 
-- conda install -c conda-forge cupy
+- `conda install -c conda-forge cupy`
 
 To install `QuTiP >5.0.0` while it is not yet released we recommend:
 
-- python -mpip install git+https://github.com/qutip/qutip.git@dev.major
+- `python -mpip install git+https://github.com/qutip/qutip.git@dev.major`
 
 Now you can safely install `qutip_cupy`
 
-- python -mpip install git+https://github.com/qutip/qutip-cupy.git
+- `python -mpip install git+https://github.com/qutip/qutip-cupy.git`
+
+Usage
+------------
+
+The main object that `qutip-cupy` provides is `CuPyDense` which is a `CuPy` based interface to store `Qobj`'s data.
+
+When working with a new `Qobj` you may proceed as follows:
+
+``` python
+import qutip 
+import qutip_cupy
+from qutip_cupy import CuPyDense
+
+qobj = qutip.Qobj( CuPyDense([2,1] ))
```

Comment:
I think one can do `qobj = qutip.Qobj([2, 1], dtype="cupyd")` and not have to import `CuPyDense`, which is possibly nicer for users.

It would be good to show some actual examples of the `QObj` being used, and then show at the end that the resulting `.data` is still a `CuPyDense` object.

We should also explain that some operations are dispatched to the GPU and return immediately before necessarily completing (e.g. multiplication) and others (e.g. expect, other operations that return scalars) block.